gremlin/README.md

```diff
@@ -25,6 +25,8 @@ their default values. See values.yaml for all available options.
 | `nodeSelector`                         | Map of node labels for pod assignment for the `gremlin` container | `{}`                                                                        |
 | `tolerations`                          | List of node taints to tolerate for the `gremlin` container    | `[]`                                                                        |
 | `affinity`                             | Map of node/pod affinities for the `gremlin` container         | `{}`                                                                        |
+| `podLabels.gremlinDaemonSet`           | Pod labels for gremlin agent daemonset | `{}` |
+| `podLabels.chao`                       | Pod labels for chao deployment | `{}` |
```

Comment:
Can we invert this so that it's inline with what we do for other attributes that apply to both objects? e.g. `gremlin.nodeSelector` and `chao.nodeSelector`.

Reply:
Can do